roles/nvidia-dgx-firmware/README.md

```diff
@@ -0,0 +1,61 @@
+# nvidia-dgx-firmware role
+
+This role makes use of the NVIDIA DGX firmware container and can be used to:
+1. Collect health & configuration diagnostics
+2. Update the firmware
+
+## Setup
+
+1) Download the latest DGX firmware container and put it in `./deepops/config/containers/dgx-firmware`. Keep the original file name. Update the role variables to reflect the version being used. 
```

Comment:
The default value is `{{ inventory_dir }}/containers/dgx-firmware`, so instead of `./deepops/config/containers/dgx-firmware` we should say `config/containers/dgx-firmware` (which isn't great either since they may not be using `config/`).

Maybe we should just default to the current working directory, which would usually be the repo directory? Then we could document that if you downloaded to `/tmp` or somewhere else, just pass i.e. `-e fw_dir=/tmp`

Reply:
Both the `old` version and the `diag` that was committed recently had the firmware container in the config (aka inventory directory as it was called in `old`). Agree that putting it elsewhere (such as the working dir, might make more sense. Then add an exception to .gitignore in case the working dir is the repo directory?

So `fw_dir` is something else - it is the remote firmware directory. Perhaps it is better to name this `remote_fw_dir` for clarity.

Reply:
I'm actually in favor of keeping it in config. That solidifies the idea that it is not ephemeral and that it is a specific configuration file used for something. If we default to somewhere outside the repo, we might hit permissions issues or it could easily get lost. If we default to working dir, same thing or they might accidentally check it in.

I'm in favor of having a seperate variable to change this directory.

Reply:
Updated in README. Keeping `config` location for now and we can always update later + add a variable (for location) if necessary.